### PR TITLE
tests: pin setuptools

### PR DESCRIPTION
**Pull request by jrcastro2**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -62,6 +62,7 @@
     'nbconvert[execute]>=4.1.0',
     'nbformat>=4.0.1',
     'tornado>=4.1,<=5.1.1',  # required by nbconvert -> jupyter-client
+    'invenio-db>=1.0.13'
```
Comment (lnielsen) [inline setup.py:65]: You'll need to also add ``[versioning]`` and remove it from the ``tests_requires``

File: setup.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
 install_requires = [
     'cchardet>=1.0.0',
-    'Flask-BabelEx>=0.9.3',
+    'Flask-BabelEx>=0.9.4',
```
Comment (lnielsen) [inline setup.py:54]: You can replace this with ``invenio-i18n>=1.3.1`` instead

**Review by lnielsen — APPROVED** [review]
In addition to the two changes, you can give it a try and change the tests requires to this:

```python 
tests_require = [
    'pytest-invenio>=1.4.3',
]
```

You might need to fix a couple of imports:

```python
from mock import ...
->
from unittest.mock import
```

**Review by ntarocco — APPROVED** [review]

**Comment by ntarocco** [comment]
CI still not happy?

**Pull request merged**